aklite-callback.sh

```diff
@@ -4,6 +4,23 @@
 # It notifies the fiotest container about the callback so that it can
 # handle everything.
 
+# First make sure this app is still installed:
+# aktualizr-lite "uninstalls" a compose-app by basically running:
+#  docker-compose stop; rm -rf /var/sota/compose-apps/fiotest
+# we can detect we were deleted by looking for that directory:
+cd $(dirname $0)
+if [ ! -d ./compose-apps/fiotest ] ; then
```

Comment:
Do we need to cd here?

Reply:
We sort of have undefined behavior from aklite for this https://github.com/foundriesio/aktualizr-lite/blob/master/src/helpers.cc#L288

I thought this trick might help if a user happened to use a non-standard location like /var/sota-foo. Not sure its worth it though?

Reply:
@ricardosalveti - what do you think about this?